### PR TITLE
added chakra/basic layout

### PR DESCRIPTION
**Pull request by MajidAliKhanQuaid**

### CODE REVIEW

File: components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  MenuDivider,
+  useDisclosure,
+  useColorModeValue,
+  Stack,
+  useColorMode,
+  Center,
+} from '@chakra-ui/react'
+import NextLink from 'next/link'
+
+const NavLink = ({ children }: { children: ReactNode }) => (
```
Comment (alisajidcs) [inline components/Navbar.tsx:23]: Move this in a separate file
Create a folder navbar, put this file in index.tsx. And place all sub components in separate files in that folder

File: components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+)
+
+export default function Nav() {
+  const { colorMode, toggleColorMode } = useColorMode()
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  return (
+    <>
+      <Box bg={useColorModeValue('gray.100', 'gray.900')} px={4}>
+        <Flex h={16} alignItems={'center'} justifyContent={'space-between'}>
+          <Box>
```
Comment (alisajidcs) [inline components/Navbar.tsx:45]: move this a separate component <HomeIcon />

File: components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            <NextLink href={'/'}>Blog with Ali</NextLink>
+          </Box>
+
+          <Flex alignItems={'center'}>
```
Comment (alisajidcs) [inline components/Navbar.tsx:49]: Move this in a separate component <NavItems />

File: package.json [hunk 4]
```diff
@@ -9,17 +9,24 @@
...
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^6.0.0",
     "eslint-plugin-react": "^7.30.0",
+    "framer-motion": "^6",
     "next": "12.1.5",
     "react": "18.0.0",
     "react-dom": "18.0.0",
-    "three": "^0.139.2"
+    "three": "^0.139.2",
+    "yarn": "^1.22.19"
```
Comment (alisajidcs) [inline package.json:29]: Why do we need this?

File: package.json [hunk 5]
```diff
@@ -9,17 +9,24 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^2.0.2",
+    "@chakra-ui/react": "^2.2.1",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
     "@typescript-eslint/parser": "^5.27.0",
+    "add": "^2.0.6",
```
Comment (alisajidcs) [inline package.json:17]: What is this for? I don't see any import for this?

File: package.json [hunk 6]
```diff
@@ -9,17 +9,24 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^2.0.2",
+    "@chakra-ui/react": "^2.2.1",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
```
Comment (alisajidcs) [inline package.json:15]: What is this for? I don't see any import for both of these..

**Review by alisajidcs — CHANGES_REQUESTED** [review]
I have suggested few tweaks. look into it.

File: package.json (hunk 6 above)
Comment (MajidAliKhanQuaid) [reply package.json:15]: these are used by chakra

File: package.json (hunk 5 above)
Comment (MajidAliKhanQuaid) [reply package.json:17]: these are REMOVED now

File: package.json (hunk 4 above)
Comment (MajidAliKhanQuaid) [reply package.json:29]: these are REMOVED now

File: components/Navbar.tsx (hunk 3 above)
Comment (MajidAliKhanQuaid) [reply components/Navbar.tsx]: Done

File: components/Navbar.tsx (hunk 2 above)
Comment (MajidAliKhanQuaid) [reply components/Navbar.tsx]: Done

**Review by alisajidcs — APPROVED** [review]

**Pull request merged**